<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginFrame</class>
 <widget class="QDialog" name="LoginFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginFrame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ColleagueRadioButton" name="radioButton_Guest">
       <property name="text">
        <string>Guest</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColleagueRadioButton" name="radioButton_Login">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Username">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColleagueTextField" name="lineEdit_Username"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Password">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColleagueTextField" name="lineEdit_Password"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ColleagueButton" name="pushButton_Ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColleagueButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColleagueRadioButton</class>
   <extends>QRadioButton</extends>
   <header>colleague_radiobutton.hpp</header>
  </customwidget>
  <customwidget>
   <class>ColleagueTextField</class>
   <extends>QLineEdit</extends>
   <header>colleague_textfield.hpp</header>
  </customwidget>
  <customwidget>
   <class>ColleagueButton</class>
   <extends>QPushButton</extends>
   <header>colleague_button.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
